portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsSegmentation.testcase

```diff
@@ -208,4 +208,71 @@ definition {
 			entryTitle = "Segments Entry");
 	}
 
+	@description = "This is a use case for LPS-199083 - Segment is not support locales in review changes."
```

Comment:
For the description, we should describe the purpose of the test instead of the title of the ticket, such as `Segment should support locales in review changes`, can you update it? Thanks.